### ISSUE TITLE
Close button - not close the app in window.

### ISSUE THREAD

**Issue by danies8** [dotnet/maui#7197]
### Description

Close button on window not close the app.
![image](https://user-images.githubusercontent.com/1079689/168476486-3f0d3b64-be12-4786-9f8d-74260bae1808.png)

### Steps to Reproduce

1.Create MAUI app.
2.Ru app.
3.Press on close button=> it didn't close the window

### Version with bug

Release Candidate 3 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

...

### Did you find any workaround?

No

### Relevant log output

```shell
No output
```

**Comment by drasticactions** [comment]
Could you add your general system info from the VS About box to your issue? I have a feeling it's probably due to whatever version of Windows you're running, since I can't reproduce this on Windows 11.

Likewise, can you try building a WinUI 3 app (without MAUI) and seeing if it does the same behavior?

**Comment by XamlTest** [comment]
Verified this issue with Visual Studio Enterprise 17.3.0 Preview 1.0 [32427.455.main]. Not repro on Windows 10 with a MAUI app.

**Comment by danies8** [comment]
It reproduced on Windows 10 with a MAUI app=> the close button behave like maximize button

**Comment by danies8** [comment]
I used Windows 10 Pro

**Comment by PureWeen** [comment]
@danies8 what version of Windows 10 are you on?

**Comment by danies8** [comment]
![image](https://user-images.githubusercontent.com/1079689/168743468-700674c2-b1cb-4912-970a-a10814798b9f.png)

**Comment by gerhard17** [comment]
I have the same problem, when I switch to <WindowsPackageType>None</WindowsPackageType>.
Then all interactions with the top app bar do not work. (Minimize, Maximize, Close and Resize)

**Comment by gerhard17** [comment]
Who needs a close box...

**Comment by danies8** [comment]
See also another issues:
https://learn.microsoft.com/en-us/answers/questions/1248238/why-the-app-is-closing-when-i-press-on-the-left-co
https://learn.microsoft.com/en-us/answers/questions/1256211/why-the-maui-app-close-when-i-press-on-left-side-o

**Comment by danies8** [comment]
Please see : https://www.screencast.com/t/r568meaiQ7 
Why the MAUI app close when I press on left side of the title bar ?
`<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="AutomationClient.MAUI.LoginPage"
             xmlns:vm="clr-namespace:AutomationClient.MAUI.ViewModels"  >
    <ContentPage.BindingContext>
        <vm:LoginViewModel/>
    </ContentPage.BindingContext>


 public partial class App : Application
    {
        public App()
        {
            InitializeComponent();

           MainPage = new LoginPage();

            
        }

      
    }`

**Comment by AnnYang01** [comment]
Verified this on Visual Studio Enterprise 17.6.0 Preview 6.0. Not repro on Windows 11 and Windows 10 pro-21H2 using below Project.
[MauiApp3.zip](https://github.com/dotnet/maui/files/11411589/MauiApp3.zip)

**Comment by eschoker** [comment]
Reproduced with the latest VS. Non of the buttons in the top left corner are working.
It was working fine until I updated VS by the way.
![maui_x_not_working](https://github.com/dotnet/maui/assets/35308242/0175f0f7-4c58-4e9c-b9df-25758e0c0f7c)

Also I noticed two other things. 

1. Some times it will make a build that will work as expected.
Some times I will build no issues and have hot reload, but if my hot reload does not work I get the red not working symbol then the top bar is not working either if I run the exe

2. I can only run the exe file if I add 

		```
<WindowsPackageType>None</WindowsPackageType>
		<UseWinUI>true</UseWinUI>
```

But then hot reload does not work any more and I even get an error that I need to deploy the project first, even if I have selected deploy from the build options.

I think we are seen a build bug or an build issue without assemblies and everything else are sympomse.
